bloodhound/ad/trusts.py

```diff
@@ -104,15 +104,14 @@ def to_output(self):
             is_transitive = not self.has_flag('NON_TRANSITIVE')
 
         # Make sure this is an int
-        trust_direction = int(self.direction)
-        trusttype_out = self.bh_trust_type.get(trusttype, 4)
+        trust_direction = self.direction_map.get(int(self.direction))
```

Comment:
This seems to still `int()` the read value directly, instead of parsing the string first? Am I missing something?

Also, I would have expected that a test needed to change for this?

Reply:
The value self.direction is an int and is correct.
With :
trust_direction = int(self.direction)
trust_direction is therefore an init too, but bloodhound is expecting a string.
That's why I use direction_map.